_templates/component/new/module.ejs.t

```diff
@@ -0,0 +1,42 @@
+---
+to: src/components/<%= name %>/index.tsx
+---
+<% if(!withExample) { -%>
+import React from "react"
+
+export function <%= name %>() {
+  return <></>
+}
+<% } else if(useState) { -%>
+import React, { useState } from "react"
+import { Button, StyleSheet, Text, View } from "react-native"
+
+type Props = Readonly<{}>
+
+export function <%= name %>(_: Props) {
+  const [count, setCount] = useState(0)
+  return (
+    <View style={styles.container}>
+      <Text>You clicked {count} times</Text>
+      <Button onPress={() => setCount(count + 1)} title="<%= name %>" />
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {},
+})
+<% } else { -%>
+import React from "react"
+import { StyleSheet, Text } from "react-native"
+
+type Props = Readonly<{}>
+
+export function <%= name %>(_: Props) {
+  return <Text style={styles.container}><%= name %></Text>
```

Comment:
I feel like the stateless function should still be wrapped in a View. I feel like I'd often generate a component without state, remove the text component, and then add a view component.

Reply:
@thispastwinter That's a great point, I think I'll implement that change. 👍